integrationTests/singleShard/block/executingMiniblocksSc_test.go

```diff
@@ -333,98 +766,94 @@ func createTxEndGame(tn *integrationTests.TestProcessorNode, round int, scAddres
 	return tx
 }
 
-func createTxJoinGame(tn *integrationTests.TestProcessorNode, joinGameVal *big.Int, round int, scAddress []byte) *transaction.Transaction {
+func createTxJoinGame(sndAddress []byte, signer crypto.SingleSigner, privKey crypto.PrivateKey, joinGameVal *big.Int, round int, scAddress []byte) *transaction.Transaction {
```

Comment:
Also here, might use TestWalletAccount as param

Reply:
did not want to break the compatibility with node. so Node can send as well. In this case, the code is more reusable.